hw/bitstream/BUILD

```diff
@@ -58,16 +67,45 @@ filegroup(
     srcs = select({
         "bitstream_skip": ["skip.bit"],
         "bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_mask_rom"],
-        "bitstream_gcp": ["@bitstreams//:bitstream_mask_rom"],
+        "bitstream_gcp_splice_rom": [":fpga_cw310_mask_rom"],
         "//conditions:default": ["@bitstreams//:bitstream_mask_rom"],
     }),
 )
 
-# TODO(lowRISC/opentitan#13603): Use `select` once we're uploading these
-# artifacts to GCP.
 filegroup(
-    name = "mask_rom_otp_dev",
-    srcs = [
-        "//hw/bitstream/vivado:fpga_cw310_mask_rom_otp_dev",
-    ],
+    name = "rom_mmi",
+    srcs = select({
+        "bitstream_skip": ["skip.bit"],
+        "bitstream_vivado": ["//hw/bitstream/vivado:rom_mmi"],
+        "//conditions:default": ["@bitstreams//:rom_mmi"],
+    }),
+)
+
+filegroup(
+    name = "otp_mmi",
+    srcs = select({
+        "bitstream_skip": ["skip.bit"],
+        "bitstream_vivado": ["//hw/bitstream/vivado:otp_mmi"],
+        "//conditions:default": ["@bitstreams//:otp_mmi"],
+    }),
+)
+
+# Build the Mask ROM and splice it into a cached bitstream.
+bitstream_splice(
+    name = "fpga_cw310_mask_rom",
```

Comment:
My only concern here is the name: `fpga_cw310_mask_rom` has the same name as an entity in `vivado/...` (same comment for the next rule).

I think this concern is alleviated by the fact this rule is private and only selectable via the filegroup above.  But if you want my opinion on the matter (:stuck_out_tongue:), I'd suggest something like `gcp_spliced_{test,mask}_rom`.

Reply:
Done. I was attempting to describing what the target is rather than how it's computed, but I admit that principle may have actually obscured the meaning of these targets.